{"id":11348,"className":"org.xml.sax.SAXParseException","methodName":"SAXParseException","signature":"\u003corg.xml.sax.SAXParseException: void SAXParseException(java.lang.String,org.xml.sax.Locator,java.lang.Exception)\u003e","javadoc":"/** \n * Wrap an existing exception in a SAXParseException. \u003cp\u003eThis constructor is especially useful when an application is creating its own exception from within a  {@link org.xml.sax.ContentHandler ContentHandler} callback, and needs to wrap an existing exception that is not asubclass of  {@link org.xml.sax.SAXException SAXException}.\u003c/p\u003e\n * @param message The error or warning message, or null touse the message from the embedded exception.\n * @param locator The locator object for the error or warning (may benull).\n * @param e Any exception.\n * @see org.xml.sax.Locator\n */\n","tf":{"freq":{"need":1,"warn":2,"link":2,"use":1,"object":1,"org":3,"applic":1,"especi":1,"exist":2,"constructor":1,"wrap":2,"may":1,"locat":3,"except":7,"error":2,"creat":1,"messag":3,"xml":3,"embed":1,"content":2,"xpars":1,"asubclass":1,"callback":1,"sax":3,"within":1,"xexcept":2,"tous":1,"pars":1,"benul":1,"handler":2},"maxFreq":7,"totalTerms":54},"loc":9}